docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -170,87 +149,225 @@ correct Google Workspace plan.
-    ![service account unique ID](../../../img/googleoidc/serviceacct-uniqueid.png)
+Click the newly-created account to view its details, and copy the Unique ID for later.
 
-  Create a new key for the service account, select JSON as the key type, and save the resulting JSON file.
-    ![service account key creation](../../../img/googleoidc/serviceacct-key.png)
-    Later, we will make this JSON available to the Teleport Auth Server via the OIDC Connector configuration, either by referencing a local file or pasting the JSON into the configuration YAML. If you plan to take the first approach, you will need to upload the JSON to the Auth Server.
+![service account unique ID](../../../img/sso/googleoidc/serviceacct-uniqueid.png)
+
+Create a new key for the service account, select JSON as the key type, and save
+the resulting JSON file.
+
+![service account key creation](../../../img/sso/googleoidc/serviceacct-key.png)
+
+Later, we will make this JSON available to the Teleport Auth Server via the
+OIDC Connector configuration, either by referencing a local file or pasting the
+JSON into the `tctl` command creating the connector. If you plan to take the
+first approach, you will need to upload the JSON to the Auth Server.
 
 <Admonition type="note">
-  Teleport requires the service account JSON to be available to all Teleport Auth Server hosts when deploying Teleport in a High Availability configuration. Unless you paste the JSON into the OIDC Connector configuration, you will need to upload the JSON file to all Auth Server hosts.
+Teleport requires the service account JSON to be available to all Teleport Auth
+Server hosts when deploying Teleport in a High Availability configuration.
+For distributed / HA or cloud deployments, we suggest providing the JSON when
+creating the connector resource.
 </Admonition>
 
 ### Configure domain-wide delegation
 
 Configure [domain-wide
-  delegation](https://admin.google.com/ac/owl/domainwidedelegation) for your
-  newly-created service account:
+delegation](https://admin.google.com/ac/owl/domainwidedelegation) for your
+newly-created service account:
 
-  Click "Add new" and add the numeric Unique ID that you've copied earlier.
-    ![domain-wide delegation](../../../img/googleoidc/domainwidedelegation.png)
+Click "Add new" and add the numeric Unique ID that you've copied earlier from
+the service account:
 
-  Add either the
-    `https://www.googleapis.com/auth/cloud-identity.groups.readonly` scope or
-    the `https://www.googleapis.com/auth/admin.directory.group.readonly` scope.
-    The scope granted to the service account will determine if Teleport will
-    fetch both direct and indirect groups or just direct groups, respectively.
+![domain-wide delegation](../../../img/sso/googleoidc/domainwidedelegation.png)
 
-## Step 3/4. Create an OIDC connector
+Add **one** of the following scopes (more info [here](#how-google-workspace-apis-differ-by-service-level)):
```

Comment:
I think this link should be more specific about what the target is pointing to, rather than using "here" (e.g., see: https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/).

Also, it looks like this is the only link to this section of the page. Would it make sense to put this information in `Details` box here instead so the reader doesn't need to jump around?

Reply:
Considering that the link points back up so what the reader should have already read and the descriptive nature of the bullets below, I think we can remove the link:

```suggestion
Add **one** of the following scopes depending on your Google Workspace:
```